### DISCUSSION TITLE
posts/systemd-limits/

### DISCUSSION THREAD

**Discussion by giscus** [aorith/aorith.github.io#4] [Announcements]
# posts/systemd-limits/

In the world of system administration and resource management, understanding how to set and test resource limits is crucial. Today, we’ll delve into systemd, a vital component in many Linux distributions, and explore how to impose limits on resources like memory and CPU. We’ll also walk through practical examples to test these limits effectively.
Setting Resource Limits In the systemd configuration, you can define resource limits for services and units. Here’s an example configuration snippet that sets memory and CPU limits for a service:

https://aorith.github.io/posts/systemd-limits/

**Comment by aorith** [comment]
Testing giscus :)
